link

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env php
+<?php
+
+require __DIR__.'/vendor/symfony/filesystem/Exception/ExceptionInterface.php';
+require __DIR__.'/vendor/symfony/filesystem/Exception/IOExceptionInterface.php';
+require __DIR__.'/vendor/symfony/filesystem/Exception/IOException.php';
+require __DIR__.'/vendor/symfony/filesystem/Filesystem.php';
```

Comment:
why explicit requires rather than requiring the autoloader ?

Reply:
Dummy copy/paste from Symfony's implementation

Reply:
The difference is that the Symfony script is usable without running `composer install`, as the require calls are in the package itself, not in `vendor`. But here, you rely on `composer install` having run already anyway (and so the autoloader will be there too).

Reply:
Ah ah.. That's so obvious.. Feel Dumb now..

Fixed in #498